src/operator/rnn.cc

```diff
@@ -427,7 +443,9 @@ NNVM_REGISTER_OP(_backward_RNN)
 .set_attr_parser(ParamParser<RNNParam>)
 .set_attr<bool>("TIsLayerOpBackward", true)
 .set_attr<nnvm::TIsBackward>("TIsBackward", true)
+#if MXNET_USE_MKLDNN == 1
```

Comment:
Same here, merge this check with the one at L450.

Reply:
Done. Thanks.